seed/static/seed/partials/data_upload_modal.html

```diff
@@ -202,6 +208,49 @@ <h4 class="modal-title" ng-switch-when="12">Error Processing Data</h4>
             </div>
         </div>
     </div>
+    <!-- Step 13: Portfolio Manager Data Import  -->
+    <div class="data_upload_steps" ng-switch-when="13">
+        <!--<form action="/pm_integration_worker/" method="post">-->
+            <div class="row">
+                <label class="control-label col-lg-3 col-sm-3" for="inputPMEmail">Email</label>
+                <div class="col-lg-8 col-sm-8">
+                    <input type="text" class="form-control" id="inputPMEmail" name="email" ng-model="pm_email" ng-init="pm_email='rdmitchell@lbl.gov'" sd-enter="console.log('What does sd-enter do?');" required>
+                </div>
+            </div>
+            <div class="row">
+                <label class="control-label col-lg-3 col-sm-3" for="inputPMUsername">Username</label>
+                <div class="col-lg-8 col-sm-8">
+                    <input type="text" class="form-control" id="inputPMUsername" name="username" ng-model="pm_username" ng-init="pm_username='SEED_City_Test'" sd-enter="console.log('What does sd-enter do?');" required>
+                </div>
+            </div>
+            <div class="row">
+                <label class="control-label col-lg-3 col-sm-3" for="inputPMPassword">Password</label>
+                <div class="col-lg-8 col-sm-8">
+                    <input type="password" class="form-control" id="inputPMPassword" name="password" ng-model="pm_password" sd-enter="console.log('What does sd-enter do?');" required>
+                </div>
+            </div>
+            <div class="row">
+                <div class="col-lg-3 col-sm-3"></div>
+                <div class="col-lg-6 col-sm-6">
+                    <button ng-click="get_pm_report_template_names(pm_email, pm_username, pm_password);">Get Report Templates</button>
+                </div>
+                <div class="col-lg-3 col-sm-3"></div>
+            </div>
+            <div class="row">
+                <label class="control-label col-lg-3 col-sm-3" for="inputPMReportTemplateName">Report Template Name</label>
+                <div class="col-lg-8 col-sm-8">
+                    <input type="text" class="form-control" id="inputPMReportTemplateName" name="template_name" ng-model="pm_template_name" ng-init="pm_template_name='SEED City Test Report'" sd-enter="console.log('What does sd-enter do?');" required>
+                </div>
+            </div>
+            <div class="row">
+                <div class="col-lg-3 col-sm-3"></div>
+                <div class="col-lg-6 col-sm-6">
+                    <button ng-click="get_pm_report(pm_email, pm_username, pm_password, pm_template_name);">Submit</button>
+                </div>
+                <div class="col-lg-3 col-sm-3"></div>
+            </div>
+        <!--</form>-->
+    </div>
```

Comment:
There is probably some small pattern that I need to grok here and all this will fall into place.  Until then, this is hacky and not working right.